subprojects/docs/src/docs/userguide/migration/upgrading_version_8.adoc

```diff
@@ -38,6 +38,21 @@ The previous step will help you identify potential problems by issuing deprecati
 [[changes_8.1]]
 == Upgrading from 8.0 and earlier
 
+=== CACHEDIR.TAG files are created in global cache directories
+Gradle now emits a `CACHEDIR.TAG` file in some global cache directories (see the release notes for details).
```

Comment:
I'm unsure if `see release notes` is a good instruction here, as I think it will break in the future. But I also don't want to just copy the list from there. Do we have a way to refer to the release notes in such a manner? Or maybe I should document it all with the rest of the cache configuration, and have this and the release notes link there?

Reply:
As suggested, this should be in the docs somewhere and linked from here. And release notes should link to here.